ts/DataGrid/DataGrid.ts

```diff
@@ -22,8 +22,9 @@
  *
  * */
 
-import type { Options, GroupedHeaderOptions } from './Options';
+import type { Options, GroupedHeaderOptions, IndividualColumnOptions } from './Options';
```

Comment:
Perhaps there is an empty space after comma so linter can return an error.
```
import type {
 Options,
 GroupedHeaderOptions,
I ndividualColumnOptions
} from './Options';
```

looks better.